<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentView</class>
 <widget class="QWidget" name="DocumentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>527</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProject">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Structure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_documentNew">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Document New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/document-new.svg</normaloff>:/images/document-new.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_folderNew">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Folder New&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/folder-new.svg</normaloff>:/images/folder-new.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_elementDel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove File/Folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/images/edit-delete.svg</normaloff>:/images/edit-delete.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::TargetMoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="targetStack">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="stackedWidget2Page1">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Targets:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="targetList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="iconSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="WaitingSpinnerWidget" name="waitSpinner" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="DebugInterface" name="tabDebug">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DebugInterface</class>
   <extends>QWidget</extends>
   <header>debuginterface.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WaitingSpinnerWidget</class>
   <extends>QWidget</extends>
   <header>waitingspinnerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>buildProject()</signal>
 </slots>
</ui>
